content/zh-cn/docs/concepts/security/_index.md

```diff
@@ -172,14 +172,14 @@ VMware vSphere | https://www.vmware.com/security/hardening-guides |
 You can define security policies using Kubernetes-native mechanisms,
 such as [NetworkPolicy](/docs/concepts/services-networking/network-policies/)
 (declarative control over network packet filtering) or
-[ValidatingAdmisisonPolicy](/docs/reference/access-authn-authz/validating-admission-policy/) (declarative restrictions on what changes
+[ValidatingAdmissionPolicy](/docs/reference/access-authn-authz/validating-admission-policy/) (declarative restrictions on what changes
 someone can make using the Kubernetes API).
 -->
 ## 策略
 
 你可以使用 Kubernetes 原生机制定义安全策略，例如
 [NetworkPolicy](/zh-cn/docs/concepts/services-networking/network-policies/)（对网络数据包过滤的声明式控制）
-或 [ValidatingAdmisisonPolicy](/zh-cn/docs/reference/access -authn-authz/validating-admission-policy/)
+或 [ValidatingAdmissionPolicy](/zh-cn/docs/reference/access -authn-authz/validating-admission-policy/)
```

Comment:
Along with typo fix in the link text, we should also remove the unnecessary space in the URL to ensure the hyperlink renders correctly. Currently, the hyperlink is rendering incorrectly on current page [refer here](https://kubernetes.io//zh-cn/docs/concepts/security/#%E7%AD%96%E7%95%A5).

```suggestion
或 [ValidatingAdmissionPolicy](/zh-cn/docs/reference/access-authn-authz/validating-admission-policy/)
```